packages/bpk-component-banner-alert/src/BpkBannerAlert.js

```diff
@@ -101,9 +120,26 @@ class BpkBannerAlert extends Component {
     }));
   }
 
+  onDismiss() {
+    if (this.props.onDismiss) {
```

Comment:
you should probably check if it's a function with `typeof`

Reply:
Yeah I also had a chat to Matt about that, and he said that the type is enforced by the proptypes anyway. If it's not a function, react will blow up